tests/integration/dbapi/sync/test_system_engine.py

```diff
@@ -68,7 +68,7 @@ def db_specs(region, attached_engine):
 
 @mark.parametrize(
     "query",
-    ["SELECT 1", "CREATE DIMENSION TABLE dummy(id INT)", "SHOW TABLES", "SHOW INDEXES"],
+    ["CREATE DIMENSION TABLE dummy(id INT)", "SHOW TABLES", "SHOW INDEXES"],
```

Comment:
Should we add a test for SELECT 1 to succeed since that's the expected behaviour now?

Reply:
Makes sense